src/sql/src/parsers/alter_parser.rs

```diff
@@ -73,6 +73,15 @@ impl<'a> ParserContext<'a> {
                     parser.peek_token()
                 )));
             }
+        } else if ParserContext::inner_consume_token(parser, "MODIFY") {
```

Comment:
Why do we need `inner_consume_token()` here? Could we use `parser.consume_token()`?
```suggestion
        } else if parser.consume_token("MODIFY") {
```

Reply:
I used parser.consume_token() at first, but it was not compatible with both upper and lower case

Reply:
Sorry, I mean `self.consume_token()`. There is no need to keep the `&mut self.parser` in L33 which prevent us using `&mut self` later.
```suggestion
        } else if self.consume_token("MODIFY") {
```

Reply:
lol, I also tried it, but because parser is a mutable reference of self.parser, self cannot be used here.

Reply:
I'm troubled by this, and I don't want to get an `inner_consume_token`